src/components/add-button/index.tsx

```diff
@@ -6,41 +6,47 @@ import { Plus } from 'react-feather';
 import { getSpace, SpaceSize } from '../space';
 import styled from 'styled-components';
 
-export interface AddPageButtonProps {
+export interface AddButtonProps {
 	onClick?: React.MouseEventHandler<HTMLElement>;
+	text: string;
+	margin?: boolean;
 }
 
-const StyledAddPageButton = styled.button`
+const StyledAddButton = styled.button`
 	position: relative;
 	box-sizing: border-box;
 	height: 60px;
 	width: 100%;
-	border: 1px solid ${Color.Grey80};
+	border: 1px solid ${Color.Grey90};
 	border-radius: 6px;
 	background-color: transparent;
-	margin: ${getSpace(SpaceSize.XS)}px;
+	margin: ${(props: AddButtonProps) => (props.margin ? `${getSpace(SpaceSize.XS)}px` : '0')};
 	display: flex;
 	align-items: center;
 	justify-content: center;
 	transition: border 0.2s;
 	user-select: none;
 
+	&:hover {
+		border: 1px solid ${Color.Grey60};
+	}
+
 	&:focus {
 		outline: none;
 	}
 
-	&:hover {
-		border: 1px solid ${Color.Grey60};
+	&:active {
+		background-color: ${Color.Grey90};
 	}
 `;
 
 const StyledIcon = styled(Plus)`
 	margin-right: ${getSpace(SpaceSize.XS)}px;
 `;
 
-export const AddPageButton: React.SFC<AddPageButtonProps> = props => (
-	<StyledAddPageButton onClick={props.onClick}>
+export const AddButton: React.SFC<AddButtonProps> = props => (
+	<StyledAddButton margin={props.margin} text={props.text} onClick={props.onClick}>
```

Comment:
Remove the text prop here and use a distinct `StyledAddButtonInterface` instead

Reply:
Just out of curiousity: Why is it better here to split the prop interfaces?

Reply:
I removed the text property and instead set the text as a children now.